packages/sw-runtime-caching/test/sw/cache-first.js

```diff
@@ -0,0 +1,33 @@
+importScripts('setup.js');
+
+describe('Test of the CacheFirst handler', function() {
+  const CACHE_NAME = location.href;
+  const REQUEST_WRAPPER = new goog.runtimeCaching.RequestWrapper({cacheName: CACHE_NAME});
+  const CACHE_FIRST = new goog.runtimeCaching.CacheFirst({requestWrapper: REQUEST_WRAPPER, waitOnCache: true});
+  const COUNTER_URL = new URL('/__echo/counter', location).href;
+
+  let cache;
+  let initialCachedResponse;
+
+  before(() => {
+    const event = new FetchEvent('fetch', {request: new Request(COUNTER_URL)});
+    return caches.delete(CACHE_NAME)
```

Comment:
This chain looks a little funky / difficult to reason with in the inidividual tests.

This seems dangerous to do just once before all tests given that you are setting a cache entry of some kind - are you expecting that value to exist  / be changed between tests? If so, I can see what breaking pretty quickly.

Could you refactor it to reset before every test and add a comment on what this chain is doing.